introduction.md

```diff
@@ -0,0 +1,3 @@
+This file is the introduction of the course
```

Comment:
Hi, I did the second commit, but I can not summit the answers in the challenges. What I missing?